lib/spack/spack/test/conftest.py

```diff
@@ -591,6 +591,16 @@ def linux_os():
     return LinuxOS(name=name, version=version)
 
 
+@pytest.fixture(scope="function")
+def debug_output():
+    saved = tty.debug_level()
+    tty.set_debug(2)
```

Comment:
Why debug level 2?  The change you made only needs level 1.

Reply:
I ask because there is a lot more output at level 2 that has to be parsed.

Reply:
FWIW.  There's already a fixture -- `ensure_debug` -- in `test_suite.py` that could be "promoted" to `conftest.py` that sets the level to 1.